client/src/components/ReviewItem.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react'
 
-const ReviewListItem = ({ title, rating, text }) => {
+const ReviewListItem = ({ title, rating, text, deleteReview, id, currentUser }) => {
+
+  const handleDeleteClick = event => {
+		event.preventDefault()
+		deleteReview(id)
+	  }
+
+    let button = ""
+    if (currentUser) {
```

Comment:
this button should only display for an admin user or for the user that created the review. Can we update to check for this?